src/rust/bitbox02-rust/src/hww/api/bitcoin/params.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::pb;
+use pb::BtcCoin;
+
+use util::bip32::HARDENED;
+
+pub struct Params {
```

Comment:
Would be nice to link to some docs like https://en.bitcoin.it/wiki/List_of_address_prefixes for example.

Also, what's the plan with btc_params.c? I assume we'll have only one at some point but they are duplicated now. I would cross reference this and btc_params.c in a comment.